conda-recipes/llvmdev/meta.yaml

```diff
@@ -20,6 +23,7 @@
 package:
   name: llvmdev
   version: {{ version }}
+  version: {{ "%s%s" % (version, VERSION_SUFFIX) }}
```

Comment:
Is sticking `_WINDOWS10_DEBUG` in the version string allowed?  I think this might make for problems in the dependency resolver.  I think we want to add this to the build string.

Reply:
This is how `llvmlite` does it 

https://github.com/numba/llvmlite/blob/master/conda-recipes/llvmlite/meta.yaml#L1

But besides that, we should remove this and increment the build number instead.

Reply:
Huh, I assumed that versions needed to be PEP 440 compliant, but I guess not?